test_cases/search.json

```diff
@@ -12,7 +12,9 @@
       },
       "expected": {
         "properties": [
-          "Brooklyn , Kings County, New York"
```

Comment:
These strings are treated differently. If the expected output is an object, the properties are compared, but if it's a string, the string is looked up in locations.json which in turn has all the properties that result is expected to have. The actual string used to lookup the location isn't used for validation. So if these tests were failing, you would just have to change a single object in locations.json and all the tests would benefit. It was a convenience feature, but if it's confusing we can pull it out.

I liked the idea of being able to search for the same location in a handful of different ways and validate results with a single object instead of having to repeat that location's properties in each of the tests.